src/components/tree/integration/peripheral-tree-converter.ts

```diff
@@ -270,16 +267,20 @@ export class PeripheralRegisterNodeConverter implements TreeResourceConverter<Pe
         const decimal = this.formatValue(resource, resource.currentValue, NumberFormat.Decimal);
         const binary = this.formatValue(resource, resource.currentValue, NumberFormat.Binary);
 
-        mds += ('| Hex &nbsp;&nbsp; | Decimal &nbsp;&nbsp; | Binary &nbsp;&nbsp; |\n');
-        mds += ('|:---|:---|:---|\n');
-        mds += (`| ${hex} &nbsp;&nbsp; | ${decimal} &nbsp;&nbsp; | ${binary} &nbsp;&nbsp; |\n\n`);
+        mds += '**Formats**\n\n';
+        mds += '| Format &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | |\n';
+        mds += '| :--- | :--- |\n';
+        mds += `| Hex | ${hex} |\n`;
+        mds += `| Decimal |  ${decimal} |\n`;
+        mds += `| Binary | ${binary} |\n`;
+        mds += ('\n_____\n\n');
 
         const children = resource.children;
         if (children.length === 0) { return mds; }
 
-        mds += ('**Fields**\n\n');
-        mds += ('| Field | &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Bit-Range | Value |\n');
-        mds += ('|:---|:---:|:---|:---|\n');
+        mds += '**Fields**\n\n';
```

Comment:
Again. wondering if we need the `Fields` title if we show headers as the next thing.